src/appkit.rs

```diff
@@ -770,37 +777,37 @@ impl NSWindow for id {
     // Getting Layout Information
 
     unsafe fn contentRectForFrameRect_styleMask_(self, windowFrame: NSRect, windowStyle: NSUInteger) -> NSRect {
-        msg_send_stret()(self, selector("contentRectForFrameRect:styleMask:"), windowFrame, windowStyle)
+        msg_send![self, contentRectForFrameRect:windowFrame styleMask:windowStyle]
     }
 
     unsafe fn frameRectForContentRect_styleMask_(self, windowContentRect: NSRect, windowStyle: NSUInteger) -> NSRect {
-        msg_send_stret()(self, selector("frameRectForContentRect:styleMask:"), windowContentRect, windowStyle)
+        msg_send![self, frameRectForContentRect:windowContentRect styleMask:windowStyle]
     }
 
     unsafe fn minFrameWidthWithTitle_styleMask_(self, windowTitle: id, windowStyle: NSUInteger) -> CGFloat {
-        msg_send()(self, selector("minFrameWidthWithTitle:styleMask:"), windowTitle, windowStyle)
+        msg_send![self, minFrameWidthWithTitle:windowTitle styleMask:windowStyle]
     }
 
     unsafe fn contentRectForFrameRect_(self, windowFrame: NSRect) -> NSRect {
-        msg_send_stret()(self, selector("contentRectForFrameRect:"), windowFrame)
+        msg_send![self, contentRectForFrameRect:windowFrame]
     }
 
     unsafe fn frameRectForContentRect_(self, windowContent: NSRect) -> NSRect {
-        msg_send_stret()(self, selector("frameRectForContentRect:"), windowContent)
+        msg_send![self, frameRectForContentRect:windowContent]
```

Comment:
Does `msg_send!` handle `msg_send_stret`? [According to your docs](http://sasheldon.com/rust-objc/objc/macro.msg_send!.html), it doesn't. :/